lib/portage/dbapi/bintree.py

```diff
@@ -887,6 +889,14 @@ def _populate_local(self, reindex=True):
         # the Packages file will not be needlessly re-written due to
         # missing digests.
         minimum_keys = self._pkgindex_keys.difference(self._pkgindex_hashes)
+
+        if "binpkg-request-signature" in self.settings.features:
```

Comment:
Merged but we should be able to do something like:
`gpkg_only = "binpkg-request-signature" in self.settings.features`
instead of if with true/false, right?